package.json

```diff
@@ -20,21 +20,21 @@
         "release:publish": "npm publish --registry=https://registry.npmjs.org"
     },
     "dependencies": {
-        "@babel/core": "~7.24.4",
-        "@babel/eslint-parser": "~7.24.1",
+        "@babel/core": "~7.24.7",
+        "@babel/eslint-parser": "~7.24.7",
         "eslint-restricted-globals": "~0.2.0",
-        "semver": "^7.6.0"
+        "semver": "^7.6.2"
     },
     "devDependencies": {
         "@lwc/eslint-plugin-lwc": "^1.8.1",
         "@salesforce/eslint-plugin-lightning": "^1.0.0",
         "eslint": "^8.57.0",
         "eslint-plugin-import": "^2.29.1",
-        "eslint-plugin-jest": "^28.2.0",
+        "eslint-plugin-jest": "~28.4.0",
```

Comment:
This is changed to `~` and is *not* the latest version of the package because in v28.5.0, they depend on `@typescript-eslint/utils@^6 ||| ^7`. Because we don't depend on that package ourselves, it resolves to v7, and v7 requires eslint 8. We still support eslint v7, so we can't do that.

I considered bumping to latest and leveraging `resolutions`, but that always feels more tedious than it's worth.